README.md

```diff
@@ -19,9 +19,14 @@ The V3 edition of the action offers:
 - stable and oldstable aliases
 - Bug Fixes (including issues around version matching and semver)
```

Comment:
As we're updating the major tag, do we need to change V3 to V4 and update the description a little to supply information about V4 changes. Also when release is rolled out we can attach a link to change log of V4 (probably there we will describe in short why we have enabled caching by default).

Reply:
I think it would be better to add a new section (`v4`) for a new major release before the `v3` section. I think we should carefully describe the breaking changes and customer benefits introduced by these changes. Also we need to update existing examples.

Reply:
I agree with this @MaksimZhukov proposal here, these are breaking changes after all